packages/manager/modules/netapp/src/dashboard/index/module.js

```diff
@@ -12,6 +12,7 @@ import '@ovh-ux/manager-billing-components';
 import component from './component';
 import routing from './routing';
 
+import './style.scss';
```

Comment:
I don't see any file style.scss in this directory, did you forgot to commit it ?